tests/main/snap-confine/task.yaml

```diff
@@ -24,7 +24,7 @@ execute: |
     if test-snapd-tools.echo hello 2>snap-confine.stderr; then
         echo "test-snapd-tools.echo should fail to run, test broken"
     fi
-    MATCH 'cannot locate the core or legacy core snap \(current symlink missing\?\):' < snap-confine.stderr
+    MATCH 'cannot locate the base snap: core: No such file or directory' < snap-confine.stderr
```

Comment:
I don't understand why we don't get the error from sc_maybe_pick_alt_base_snap () - it calls die() with a different error if core/current is missing. What am *I* missing :) ?

Reply:
Some brilliant insight there Mvo

Reply:
The error message is wrong, the one you were expecting to be used is actually reported.